Formula/metabase.rb

```diff
@@ -21,6 +21,8 @@ class Metabase < Formula
   def install
     if build.head?
       Language::Node.setup_npm_environment
+      # help yarn find npm's node-gyp binary inside superenv
+      ENV.prepend_path "PATH", "#{Formula["node"].opt_libexec}/lib/node_modules/npm/bin/node-gyp-bin"
```

Comment:
No this could be done, but the underlying issue is that yarn auto-detects it's global installation prefix as the bin folder inside node's Cellar. If you for example globally install webpack with `yarn global add webpack`, yarn will put a symlink into `HOMEBREW_PREFIX/Cellar/node/8.2.1/bin/webpack`, which is imho harmful (nothing should manipulate another formulas cellar).

A fix would be to run `yarn config set prefix HOMEBREW_PREFIX`, but this can't be done in a post_install step because the sandbox will block write request to `~/.yarnrc` if not run with `--no-sandbox`.